### ISSUE TITLE
A11y_AzureDataStudioAAE_Sql Instance 1_Compute + Storage_Usability: Maximum and minimum value for spin button is not provided

### ISSUE THREAD

**Issue by krrish2021** [microsoft/azuredatastudio#15768]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yUsable;#A11yTCS;#A11ySev4;#SH_AzureDataStudio_AAE_Jun2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#AINotInScope;

### Environment Details:
Application Name: Azure Data Studio-AAE
Windows Version: Win10

### Repro Steps:

1. Launch the application-Azure Data Studio-Insiders and install extension Azure arc and Azure data ClI
2. Connect to Azure Arc Controller
3. Tab till Azure Arc Controller tree and tab to the node present under it
4. Tab to child node "Sqlinstance1" and open manage and tab to "Compute + Storage" page
5. Tab to spin button and check its maximum and minimum entry value

### Actual:
Maximum and minimum value for spin button control is not provided.

Similar issue is observed in the scenario given below:

1. PG1_Compute + Storage

### Expected:
Maximum and minimum value for spin button control should be provided.

### User Impact:
Users will get confused if the spin button do not have maximum and minimum entry value.

**Comment by Charles-Gagnon** [comment]
@krrish2021 I tried adding the `aria-valuemin` to the control but NVDA doesn't seem to read it, and issues I've found have all been for slider bars.

Can you link to documentation for min/max values for input spinners indicating they should be read out? Given that the control by default has no indication of a min/max it seems to me that there isn't any extra need to indicate that to screen readers or otherwise. The control itself will handle enforcing those limits regardless of any assistive technologies being used (or lack thereof)

I'm going to close this now since it doesn't seem like there's anything we can do. If you have any other suggestions for what we should be doing here let me know and we can reopen this.

**Comment by Charles-Gagnon** [comment]
Chatted with @alanrenmsft - this seems to be an NVDA issue. The input controls have a min value defined but NVDA doesn't read it properly. Narrator does though. 

(there isn't a max value here so that doesn't really apply)

**Comment by krrish2021** [comment]
Closing as not repro
![Screenshot (183)](https://user-images.githubusercontent.com/85863543/150962774-db9eb74e-4c3e-4b97-8611-9ba92c1e7b05.png)
